gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -207,8 +207,11 @@ class {{ service.name }}RestTransport({{ service.name }}Transport):
         url += '?{}'.format('&'.join(query_params)).replace(' ', '+')
 
         # Send the request
+        headers = dict(metadata)
+        headers['Content-Type'] = 'application/json'
```

Comment:
Naive question: this is not `x-goog-api-client`. Where does the name `content-type` come from? Is there documentation somewhere describing it?

Reply:
The `Content-Type` header is part of HTTP standard: https://datatracker.ietf.org/doc/html/rfc2616#section-14.17

Reply:
@software-dov See also the internal REGAPIC gRPC transcoding design doc, where we call this out explicitly (relatively recent addition, since we found some RPCs fail without it)